src/registrar/templates/portfolio_requests.html

```diff
@@ -14,12 +14,12 @@
 {% endblock %}
 
 <div id="main-content">
-    <h1 id="domain-requests-header">Domain requests</h1>
+    <h1 id="domain-requests-header" class="margin-bottom-1">Domain requests</h1>
     <div class="grid-row grid-gap">
 
         {% if has_edit_request_portfolio_permission %}
             <div class="mobile:grid-col-12 tablet:grid-col-6">
-                <p class="margin-y-0">Domain requests can only be modified by the person who created the request.</p>
+                <p class="margin-y-0 maxwidth-386">Domain requests can only be modified by the person who created the request.</p>
```

Comment:
**question** why are setting a width to a hard coded number inside of the html? Wouldn't a more scalable approach be to add this into an scss file and then just call the correct value in this paragraph?

Reply:
@abroddrick this exact width was not available in usdws. And so I created a new class in the base.scss file for a max width of 386. Should I updated the class name to be something else?

Reply:
Im fine with it not being the exact width of 386 and instead using something more scalable that is close to 386, like if there's a #-column class or something. Ideally the width of the paragraph more or less lines up like this.
![Screenshot 2024-11-20 at 9 23 31 AM](https://github.com/user-attachments/assets/d8c8f638-d69b-457e-993a-487a3a41892e)

Reply:
@gabydisarli I updated the class! let me know what you think

Reply:
This looks great!

Reply:
Love it!